pkg/gui/controllers/status_controller.go

```diff
@@ -33,21 +33,24 @@ func NewStatusController(
 func (self *StatusController) GetKeybindings(opts types.KeybindingsOpts) []*types.Binding {
 	bindings := []*types.Binding{
 		{
-			Key:         opts.GetKey(opts.Config.Universal.OpenFile),
-			Handler:     self.openConfig,
-			Description: self.c.Tr.OpenConfig,
-			Tooltip:     self.c.Tr.OpenFileTooltip,
+			Key:             opts.GetKey(opts.Config.Universal.OpenFile),
+			Handler:         self.openConfig,
+			Description:     self.c.Tr.OpenConfig,
+			Tooltip:         self.c.Tr.OpenFileTooltip,
+			DisplayOnScreen: true,
```

Comment:
I don't even remember why we still have the 'o' command here at all, but I don't think this should be shown in the status bar. They should use 'e', it's better for most people.

Instead, I would suggest to include "enter" for "switch to recent repo". I use this quite often, I find it more convenient to type `1 enter` than to type `<c-r>`, mostly because I can never remember in which panels the latter is available.